src/pages/gateway/transforms/rename.md

```diff
@@ -18,7 +18,7 @@ keywords:
 
 <InlineAlert variant="info" slots="text"/>
 
-You can use [RegEx flags] to enable the use of regular expressions when renaming using this transform. For example, you could use the key-value pair `field: api(.*)` in the `from` object to rename any field of the corresponding type that begins with "api".
+You can use regular expression flags to enable the use of regular expressions when renaming using this transform. For example, you could use the key-value pair `field: api(.*)` in the `from` object to rename any field of the corresponding type that begins with "api".
```

Comment:
This appears to have broken a link. We should ensure that the logic is only replacing alphanumeric characters.

Reply:
It was my fault, I removed the square brackets manually because I thought they were the reference to a link definition which I couldn't find in the file.

Reply:
ah, ok. yeah, i'm removing all "link definition" style links as part of https://github.com/AdobeDocs/graphql-mesh-gateway/pull/218